helpers/wrapper/contracts/Wrapper.sol

```diff
@@ -123,14 +125,14 @@ contract Wrapper {
       wethContract.withdraw(_makerAmount);
       // Transfer ether to the user.
       msg.sender.transfer(_makerAmount);
+    } else if (_makerToken != address(0)){
```

Comment:
coverage is down on this branch: only 1 branch has been tested

Reply:
cool, on it.

Reply:
![Screen Shot 2019-08-15 at 10 48 10 AM](https://user-images.githubusercontent.com/3590816/63103001-3473cc00-bf4a-11e9-9dbe-f02faee64f7a.png)